File: Algo-Review/01-LinearSearch/linearSearch.java
public class linearSearch{
    public static void main(String args[]){
        int[] array = {53, 13, 64, 34, 12, 63, 53};
        int index = linear_search(array, 34);

        if(index != -1){
            System.out.println("Element found at index: " + index);
        }
        else {
            System.out.println("Element not found");
        }
    }

    private static int linear_search(int[] array, int value){
        for(int i = 0; i<array.length; i++){
            if(array[i]==value){
                return i;
            }
        }
        return -1;
    }
}

File: Algo-Review/01-LinearSearch/linearSearch.py
def linearSearch(arr, val):
    for i in range(0, len(arr)):
        if arr[i] == val:
            return i 
    return -1
  
def main():
    arr = [53, 13, 64, 34, 12, 63, 91]
    val = 34
    index = linearSearch(arr, val)
    if (index != -1):
        print('Element found at index ', index)
    else:
        print('Element not found ')
     

if __name__ == "__main__":
    main()